package.json

```diff
@@ -20,12 +20,12 @@
     "@ckeditor/ckeditor5-block-quote": "~31.0.0",
     "@ckeditor/ckeditor5-build-balloon": "~31.0.0",
     "@ckeditor/ckeditor5-core": "^31.0.0",
-    "@ckeditor/ckeditor5-dev-utils": "~25.4.5",
-    "@ckeditor/ckeditor5-dev-webpack-plugin": "~25.4.5",
-    "@ckeditor/ckeditor5-editor-balloon": "~31.0.0",
+    "@ckeditor/ckeditor5-dev-utils": "~26.2.1",
+    "@ckeditor/ckeditor5-dev-webpack-plugin": "~26.2.1",
+    "@ckeditor/ckeditor5-editor-balloon": "~31.1.0",
     "@ckeditor/ckeditor5-essentials": "~31.0.0",
     "@ckeditor/ckeditor5-heading": "~31.0.0",
-    "@ckeditor/ckeditor5-link": "~31.0.0",
```

Comment:
some packages are now at 31.1, others at 31 – intended?

Reply:
i only added the ones that the bump already created. Should i update them all before the bump creates one?

Reply:
added the rest as it works fine